internal/tests/api/workers/worker_test.go

```diff
@@ -0,0 +1,55 @@
+package workers_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/boundary/api/workers"
+	"github.com/hashicorp/boundary/internal/daemon/controller"
+	"github.com/hashicorp/boundary/internal/daemon/worker"
+	"github.com/hashicorp/boundary/internal/types/scope"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWorkerTagsASD(t *testing.T) {
+	assert, require := assert.New(t), require.New(t)
+	tc := controller.NewTestController(t, nil)
+	defer tc.Shutdown()
+
+	client := tc.Client()
+	token := tc.Token()
+	client.SetToken(token.Token)
+
+	tw := worker.NewTestWorker(t, &worker.TestWorkerOpts{
+		WorkerAuthKms:    tc.Config().WorkerAuthKms,
+		InitialUpstreams: tc.ClusterAddrs()})
```

Comment:
Just a small gofmt thing:
```suggestion
		InitialUpstreams: tc.ClusterAddrs(),
	})
```